ddl/ddl_test.go

```diff
@@ -690,6 +697,9 @@ func TestReorg(t *testing.T) {
 }
 
 func TestCancelJobs(t *testing.T) {
+	if variable.EnableConcurrentDDL.Load() {
```

Comment:
So could we add UT for this logic?

Reply:
this UT is only used for the old framework, you can see that it pushes the job into the queue directly and try to cancel it.

we have many other UT for cancel tests in `cancel_test.go`.

Reply:
Got it.